scripts/fetch_ci_binaries.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+"""Fetch executable binary artifacts from the latest Circle CI workflow on main"""
+
+import os
+import sys
+from typing import List, Optional, Tuple
+import requests
+
+HTTP_TIMEOUT = 5
+
+DEVNET_CI_URL = (
+    "https://circleci.com/api/v1.1/project/github/0xSpaceShard/starknet-devnet-rs"
+)
+
+BINARY_BUILD_JOB_PREFIX = "binary-build-"
+
+
+def get_artifact_info(job: dict) -> Optional[Tuple[str, str]]:
+    """
+    Returns the artifact (URL, name) corresponding to job object.
+    Returns None if job not a binary build.
+    """
+    if job["job_name"].startswith("binary-build-"):
+        job_id = job["job_id"]
+        job_name = job["job_name"]
+        artifact_name = (
+            job_name.replace(BINARY_BUILD_JOB_PREFIX, "starknet-devnet-") + ".tar.gz"
+        )
+        return (
+            f"https://output.circle-artifacts.com/output/job/{job_id}/{artifact_name}",
+            artifact_name,
+        )
+
+    raise ValueError(f"Invalid job: {job}")
+
+
+def write_artifacts(workflow_id: str, artifact_infos: List[Tuple[str, str]]):
+    """Write artifacts on disk"""
+    os.mkdir(workflow_id)
```

Comment:
I'll need to test this script on Mac and I need to install python, stay tuned.

Reply:
Thanks for finding the bug, fixed it in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/423/commits/022e4b4e72d9065a809433e9231007b3eca46d7e